### PR TITLE
feat(mocks): create personas for users with different account opening states

### PR DESCRIPTION
**Pull request by abuecker**

### CODE REVIEW

**Comment by abuecker** [comment]
@zrangerz @johntowell @olmstead please review.  Thanks!

**Comment by johntowell** [comment]
I think we should load these personas into the database and then develop with the full stack.

File: server/personas/jane.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Jane has completed the forms, but has no accounts.
+ */
+export default function jane(profile) {
+  return {
+    ...profile,
+    firstName: 'Jane',
+    accounts: []
```
Comment (abuecker) [inline server/personas/jane.js]: Good idea.  Added.

**Comment by abuecker** [comment]
👍  using the personas in the full stack.

**Pull request merged**